### DISCUSSION TITLE
Unable to get REPL on MIMXRT1064-EVK

### DISCUSSION THREAD

**Discussion by ccie38216** [micropython/micropython#14053] [General]
So i got the 1064 EVK board in and i've uploaded the HEX image and i'm unable to get an REPL prompt via USB. The MCUXpresso SDK examples work just fine but for some reason I cannot get use MicroPython... Any suggestions?

**Comment by robert-hh** [comment]
Which USB port do you use for REPL. The MicroPython interface is U9 or U11 (the ones next to the Ethernet port).
Which board version to you use?

**Comment by ccie38216** [comment]
Based on documentation of cpython I was under the impression that the USB Debug port is used on the 1064 to access repl as uart0 is connected to the lpc4322. None the less, it is the only port that presents a COM port to my PC. 

The USB-OTG (U9) and USB-Host (U11) port does not present a serial port when plugged into the pc.

I've tried older versions of circuit python as well and same result. The date code on my board is 2238, REV-A

**Reply by robert-hh** [reply]
I would expect that too. There was a PR https://github.com/micropython/micropython/pull/10885 that ought to fix the MIMXRT1064-EVK configuration. Maybe @iabdalkader could chime in on that topic. I do not have a board here for testing.

**Reply by iabdalkader** [reply]
That PR fixed the flash config, it didn't touch USB. I don't have the MIMXRT1064-EVK anymore, I only have a MIMXRT1060-EVKB I can test it if you like. I remember I needed I change the default jumpers, isolate the LPC, and switch the power to OTG to get things working. But maybe I did this for debugging, not sure.
